atecc608a/main.c

```diff
@@ -443,7 +406,7 @@ void print_device_info()
     atecc608a_print_serial_number();
     atecc608a_print_config_zone();
     atecc608a_print_locked_zones();
-    printf("\nPrivate key slot in use: %lu, public: %lu\n",
+    printf("\nPrivate key slot in use: %llu, public: %llu\n",
```

Comment:
Minor: `atecc608a_private_key_slot` and `atecc608a_public_key_slot` are `uint64_t`, not `unsigned long long`. This is minor because this code is very unlikely to run on a platform where these two types are different. However, it would be better to be consistent. Note that since slot numbers in this particular driver (and in any driver based on the same model where the secure element has a static array of slots) are small, you could also use `%u` and `(unsigned) xxx_slot`.